frontend/components/slot/viz/SlotVizTooltip.vue

```diff
@@ -300,6 +300,15 @@ const data = computed(() => {
         align-items: flex-start;
         word-wrap: nowrap;
         white-space: nowrap;
+        .validators {
+          display: grid;
+          grid-template-columns: 1fr 1fr 1fr;
+          gap: var(--padding-tiny);
+          span:last-child {
+            grid-column: span 3;
+            text-align: left
+          }
+        }
```

Comment:
**Bug:** This change causes exactly two validators to be written in a column.

**Current staging**
![image](https://github.com/gobitfly/beaconchain/assets/114066135/9edf9572-2307-4d0e-a0d5-468de98dd2cb)

**Your branch**
![image](https://github.com/gobitfly/beaconchain/assets/114066135/3c3e29c5-db49-45b2-8645-73bb76fcfa29)

There is no need for two validators to be placed below each other. If there are only two validators, it should look like current staging.

It also looks odd when there are 5 validators for example:
![image](https://github.com/gobitfly/beaconchain/assets/114066135/fee3faea-d5bc-42f2-83ab-0842d1943b41)

Reply:
@marcel-bitfly , I think we should not use a grid for this. Here is a screenshot showing a tooltip using your latest commit (a26ec4b5b0a44a053a1381a95e9d23f1f8117186):
![image](https://github.com/gobitfly/beaconchain/assets/114066135/50d4328c-7d50-4fee-bcb2-75429d50219f)
As you can see, the lower validator line is not aligned correctly, there is an empty space at the beginning.

Shouldn't it suffice to simply allow wrapping for this element? I made a quick test with this code:
```css
.validators {
  text-align: left;
  white-space: wrap;
}
```
and it looks good. Is there anything against that?

Reply:
it does not work for popups with containing `...and XX more`

But it could stay as is:
https://bitfly1.atlassian.net/browse/BIDS-3203?focusedCommentId=16465